.envrc.nix

```diff
@@ -1 +1,2 @@
 use nix
+export PATH="./node_modules/.bin:$PATH"
```

Comment:
is this preferred over just using `npx <thing>` to run thing?

Reply:
☁️ Make it an absolute path?

Reply:
@allenap ahh, good catch!

@avh4 I prefer it so that I can just run things "normally"